client/src/partials/patient_invoice/patientInvoice.js

```diff
@@ -155,6 +159,22 @@ function PatientInvoiceController(Patients, PatientInvoices, PatientInvoiceForm,
 
     if (angular.isDefined(vm.services) && vm.services.length) {
       vm.Invoice.setService(vm.services[SERVICE_INDEX]);
+      vm.selectedService = vm.services[SERVICE_INDEX];
+    }
+  }
+
+  function setService (service_id) {
```

Comment:
I think the invoice service is 'set' because it is bound to the `ng-model`. What this function is actually doing is updating the description - it would be best if the function name represented this.